sdk/communication/Azure.ResourceManager.Communication/tests/Infrastructure/CommunicationManagementClientLiveTestBase.cs

```diff
@@ -96,43 +96,43 @@ internal async Task<SenderUsernameResource> CreateDefaultSenderUsernameResource(
             return senderUsername;
         }
 
-        internal async Task<SuppressionListResource> CreateDefaultSuppressionListResource(CommunicationDomainResource domain, string listName)
-        {
-            var id = Recording.Random.NewGuid().ToString();
-
-            SuppressionListResourceData data = new SuppressionListResourceData
-            {
-                 ListName = listName
-            };
-
-            ArmOperation<SuppressionListResource> suppressionListOp =
-                await domain.GetSuppressionListResources().CreateOrUpdateAsync(WaitUntil.Completed, id, data);
-            SuppressionListResource suppressionList = suppressionListOp.Value;
-            return suppressionList;
-        }
-
-        internal async Task<SuppressionListAddressResource> CreateDefaultSuppressionListAddressResource(
-            SuppressionListResource suppressionList,
-            string email,
-            string firstName = default,
-            string lastName = default,
-            string notes = default)
-        {
-            var id = Recording.Random.NewGuid().ToString();
-
-            SuppressionListAddressResourceData data = new SuppressionListAddressResourceData
-            {
-                Email = email,
-                FirstName = firstName,
-                LastName = lastName,
-                Notes = notes
-            };
-
-            ArmOperation<SuppressionListAddressResource> suppressionListAddressOp =
-                await suppressionList.GetSuppressionListAddressResources().CreateOrUpdateAsync(WaitUntil.Completed, id, data);
-            SuppressionListAddressResource suppressionListAddress = suppressionListAddressOp.Value;
-            return suppressionListAddress;
-        }
+        //internal async Task<SuppressionListResource> CreateDefaultSuppressionListResource(CommunicationDomainResource domain, string listName)
```

Comment:
why is this commented out? if not needed, can we just remove it?

Reply:
We would be removing work that someone did and has need of for a future version. They just happened to release a preview sdk before this stable release

Reply:
The test can be recovered from the git history as well.
Either way is fine.